roles/parse_c2r_report/README.md

```diff
@@ -1,38 +1,14 @@
-Role Name
+Parse C2R Report
 =========
```

Comment:
```suggestion
Parse C2R Report
==============
```

Reply:
Funny enough, when you try to suggest the change through here, the editor show you correctly that you are filling the `=` up to the name length, but when you submit the suggestion, the diff seems wrong and still missing some `=`